database/migration/src/main/resources/changesets/set_table_role.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog objectQuotingStrategy="QUOTE_ALL_OBJECTS"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <!-- The 'sharedCromwellDbRole' can be set via JAVA_OPTS if desired (eg -DsharedCromwellDbRole=...). -->
```

Comment:
Do you envision this flag being added on both WORKFLOWS and CROMWELL_RUNNER? Or just WORKFLOWS?

I imagine just WORKFLOWS because if CROMWELL_RUNNER has this flag, it will always change the metadata tables to be owned by its own user - is that the right way to think about it?

Reply:
Updated now to only affect the metadata tables. Leaving us open to having another flag in the future to set the ownership of the "working db" tables